test/lib/Benchmark.js

```diff
@@ -2,12 +2,22 @@
 "use strict";
 
 const {Ci, Cu} = require("chrome");
+const prefs = require("sdk/preferences/service");
+const self = require("sdk/self");
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 
 XPCOMUtils.defineLazyModuleGetter(this, "Task",
                                   "resource://gre/modules/Task.jsm");
 
+/*
+ * Disable "Async Stacks" to make the benchmark yield more stable results
+ *
+ * See more detail at
+ *  https://developer.mozilla.org/en-US/docs/Mozilla/Benchmarking
+ */
+Services.prefs.setBoolPref("javascript.options.asyncstack", false);
+
```

Comment:
Presumably, this only works if javascript.options.asyncstack is checked at runtime rather than startup time.  Is there some reason not to just put this in benchmark-prefs.json ?